<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="depth_controller" type="depth_controller_node" name="DepthControllerNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find depth_controller)/config/config_depth_controller.yaml"/>
</node>

</launch>
